main.go

```diff
@@ -409,8 +444,10 @@ func main() {
 }
 
 func recycleIPTables(nw ip.IP4Net, lease *subnet.Lease) error {
-	prevNetwork := ReadCIDRFromSubnetFile(opts.subnetFile, "FLANNEL_NETWORK")
+	prevNetworks := ReadCIDRsFromSubnetFile(opts.subnetFile, "FLANNEL_NETWORK")
 	prevSubnet := ReadCIDRFromSubnetFile(opts.subnetFile, "FLANNEL_SUBNET")
+
+	prevNetwork := ip.GetOvernet(&prevSubnet, prevNetworks)
```

Comment:
`ReadCIDRFromSubnetFile` is also used in `newSubnetManager`.
I tried using only one function but I found that it made the code less readable because when we read `FLANNEL_SUBNET`, we have to check whether there is exactly 1 subnet in the calling code while here it is encapsulated in the function.

I agree about `getOvernet`and the name is not clear. I inlined the code instead.